net/safe-search-bing/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2018 Gregory L. Dietsche (Gregory.Dietsche@cuw.edu)
+# This is free software, licensed under the MIT License
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=safe-search-bing
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=4
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Gregory L. Dietsche <Gregory.Dietsche@cuw.edu>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/safe-search-bing
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=Block explicit images, videos, and websites from Bing Search results.
+	PKGARCH:=all
+endef
+
+define Package/safe-search-bing/description
+Prevent adult content from showing up in Bing search results by forcing all
+devices on your network to use Bing's Safe Search IP addresses.
+For more information please see:
+https://help.bing.microsoft.com/#apex/18/en-US/10003/0
+endef
+
+define Build/Compile
+endef
+
+define Package/safe-search-bing/install
+	$(INSTALL_DIR) $(1)/etc/uci-defaults
+	$(CP) ./files/safe-search-bing.defaults $(1)/etc/uci-defaults/safe-search-bing
+
+	$(INSTALL_DIR) $(1)/usr/share/safe-search-bing
+	$(CP) ./files/hosts $(1)/usr/share/safe-search-bing/
+	chmod -R ugo+rX $(1)/usr/share/safe-search-bing/
+endef
+
+define Package/safe-search-bing/prerm
+#!/bin/sh
+uci del_list dhcp.@dnsmasq[0].addnhosts=/usr/share/safe-search-bing
```

Comment:
this looks rather specific to dnsmasq?  You probably want to _at least_ mention that in the description, if not in the DEPENDS

Reply:
Prior to sending in this patch, I tried setting DEPENDS:=dnsmasq . At least with the ar71xxx SDK, it doesn't work when trying to build a firmware image. The reason is that settings that depends pulls in dnsmasq-full for some reason. Am I using DEPENDS incorrectly?

    * check_data_file_clashes: Package dnsmasq wants to install file /data/ace/greg/documents/programming/genesis39/firmware/dynamic-files/openwrt-imagebuilder-18.06.1-ar71xx-generic.Linux-x86_64/build_dir/target-mips_24kc_musl/root-ar71xx/usr/share/dnsmasq/rfc6761.conf
        But that file is already provided by package  * dnsmasq-full
    * opkg_install_cmd: Cannot install package dnsmasq.

Reply:
I've done some checking, and typically packages that change dnsmasq configuration don't list it in their depends. Some examples include: simple-adblock, kadnode, banhosts, unbound, etc...

With that in mind, I've updated my patch to refer to dnsmasq in the description of each package.